docs/upgrade/UPGRADE-v7.3.2-dev.md

```diff
@@ -59,4 +59,87 @@ There you can find links to upgrade notes for other versions too.
     - restart you Nginx for the changes to take effect
         - for example, run `docker-compose up -d --force-recreate webserver` to restart the container in Docker Compose
 
-[shopsys/framework]: https://github.com/shopsys/framework
\ No newline at end of file
+### Frontend changes
```

Comment:
We don't use this section only sections shown [here](https://github.com/shopsys/shopsys/blob/master/docs/contributing/guidelines-for-writing-upgrade.md#section-shopsysframework). Please place it to Application section

Reply:
Moved to appliaction section